### PR TITLE
Add support for params

### PR DESCRIPTION
**Pull request by jcenturion**
Add support for `wt create index.js --name sample --param param=value`.

### CODE REVIEW

File: bin/create.js [hunk 1]
```diff
@@ -33,6 +33,14 @@ module.exports = Cli.createCommand('create', {
                 metavar: 'KEY=VALUE',
                 type: 'string',
             },
+            'param': {
+                action: 'append',
+                defaultValue: [],
+                description: 'Param(s) exposed to your code as `params` on the webtask context object. The properties will be signed and protected from interference but not encrypted.',
```
Comment (ggoodman) [inline bin/create.js]: Nice description.

**Comment by ggoodman** [comment]
LGTM.

**Comment by ggoodman** [comment]
@jcenturion can you publish this please?

**Comment by jcenturion** [comment]
Done :)

**Pull request merged**